discovery-provider/src/queries/get_db_seed_restore_status.py

```diff
@@ -0,0 +1,33 @@
+import sqlalchemy
+from src.utils.config import shared_config
+from src.utils.db_session import get_db_read_replica
+
+env = shared_config["discprov"]["env"]
+
+# environment to db file md5 hash lookup, these are constants and will not change
+env_to_hash_lookup = {"prod": "c785fc03cfc9cca3e8a48226b9cd424b"}
+
+
+def get_db_seed_restore_status():
+    db = get_db_read_replica()
+    with db.scoped_session() as session:
+        db_seed_restore_status_query = sqlalchemy.text(
+            """
+            SELECT md5(cast(array_agg(sorted_hashes."str" ORDER BY sorted_hashes."str" ASC) as text)) FROM (
```

Comment:
is the theory that this is a good enough proxy for the holes that might be in between users in tables that SPs have?

Reply:
exactly. we know this version of the db has all the data and nothing should change going forward since the dates and sha are known ahead of time